test/unit/src/core/Raycaster.tests.js

```diff
@@ -7,6 +7,9 @@ import { Raycaster } from '../../../../src/core/Raycaster';
 import { Vector3 } from '../../../../src/math/Vector3';
 import { Mesh } from '../../../../src/objects/Mesh';
 import { SphereGeometry } from '../../../../src/geometries/SphereGeometry';
+import { Geometry } from '../../../../src/core/Geometry';
```

Comment:
Please avoid the usage of `Geometry`. `BufferGeometry` should be your first choice. Use this pattern:
```js
var points = [ new Vector3( -2, -10, -5 ), new Vector3( -2, 10, -5 ) ];
var geometry = new BufferGeometry().setFromPoints( points );
```

Reply:
Fixed in 101dc18ef4a691ad77ecaf0045ffd00ed8d3418d.

Definitely more compact. :blush:

I think I just got the idea of using `Geometry` looking for how other `Line`s where created in `test/unit/src/core/`, and found just one example:

https://github.com/mrdoob/three.js/blob/fd8b70cf67f4a1ee49817b13bc2deb59f5369fb6/test/unit/src/core/BufferGeometry.tests.js#L364-L377